assets/blocks/course-actions-block/course-actions/course-actions-edit.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+const innerBlocksTemplate = [
```

Comment:
Why do we need 3 blocks instead of 1?

Reply:
We do, otherwise the blocks won't render on the front-end because they won't exist on the page. So I opted to hide their UI in the editor instead. Not ideal, but it gets the job done.